gplugins/lumerical/tests/test_lumerical_utils.py

```diff
@@ -0,0 +1,267 @@
+from gdsfactory.config import logger
+from gdsfactory.technology.layer_stack import LayerLevel, LayerStack
+
+from gplugins.lumerical.config import DEBUG_LUMERICAL, ENABLE_DOPING
+from gplugins.lumerical.utils import layerstack_to_lbr
+
+
+def test_layerstack_to_lbr():
+    """
+    Ensure process file generated from function is importable in MODE, FDTD, and CHARGE.
+    """
+    # Inputs
+    layer_map = {
+        "si": "Si (Silicon) - Palik",
+        "sio2": "SiO2 (Glass) - Palik",
+        "sin": "Si3N4 (Silicon Nitride) - Phillip",
+        "TiN": "TiN - Palik",
+        "Aluminum": "Al (Aluminium) Palik",
+    }
+
+    # Create layerstack
```

Comment:
is this common? can it be moved to a different file and imported? just so long

Reply:
Removed as much unnecessary code as possible. Layerstack definition is custom here.